app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -115,7 +116,7 @@ public void alreadyRevoked() {
     public void testEmptyList() {
         bc.addBlock(_block);
         Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, owner, sequenceNumber+1, ownHash,
```

Comment:
You should not be able to give a sequencenumber to the constructor upon creating a block